README_production.rst

```diff
@@ -0,0 +1,285 @@
+Production Deployment
+=====================
+
+This README provides documentation to help deploy Yaksh in a production
+environment. If you wish to take Yaksh on a trial run, here is a
+[Quickstart Guide]
+(https://github.com/FOSSEE/online\_test/blob/master/README.md)
+
+###################
+Deploying Locally
+###################
+
+Follow these steps to deploy locally on the server. For deployment instructions using Docker see :ref:`deploying-multiple-dockers`.
+
+Pre-Requisite
+^^^^^^^^^^^^^
+
+1. Ensure `pip <https://pip.pypa.io/en/latest/installing.html>`__ is
+   installed
+2. Install dependencies using pip install -r requirements.txt
+3. Install MySql Server
+4. Install Python MySql support
+5. Install Apache Server for deployment
+
+1. Create a database named ``yaksh`` by following the steps below
+
+   ::
+
+       $> mysql -u root -p    
+       $> mysql> create database yaksh
+
+2. Add a user named ``yaksh_user`` and give access to it on the database
+   ``yaksh`` by following the steps below
+
+   ::
+
+      mysql> grant usage on yaksh to yaksh_user@localhost identified
+      by 'mysecretpassword';
+
+      mysql> grant all privileges on yaksh to yaksh_user@localhost;
+
+3. Add ``DATABASE_PASSWORD = 'mysecretpassword'`` and
+   ``DATABASE_USER = 'yaksh_user'`` to online\_test/settings.py
+
+To deploy this app follow the steps below:
+
+1.  Clone this repository and cd to the cloned repo. 
+
+   ::
+
+       $ git clone https://github.com/FOSSEE/online\_test.git
+
+2.  Run:
+
+   ::
+
+       python manage.py makemigrations yaksh
+
+       python manage.py migrate yaksh 
+
+4.  First run the python server provided. This ensures that the code is
+    executed in a safe environment. Do this like so:
+
+    ::
+
+        $ sudo python yaksh/code_server.py
```

Comment:
This will not work. We need to add `sudo python -m yaksh.code_server` for python 2 `sudo python -m yaksh.code_server` for python 3.